docs/docs/guides/tracking/feedback.md

```diff
@@ -181,29 +221,57 @@ Additionally, you can use call() method to execute the operation and retrieve th
   </TabItem>
 </Tabs>
 
-## Human annotation
+## Add human annotations
+
+Human annotations are supported in the Weave UI. To make human annotations, you must first create a Human Annotation scorer using either the [UI](#create-a-human-annotation-scorer-in-the-ui) or the [API](#create-a-human-annotation-scorer-using-the-api). Then, you can [use the scorer in the UI to make annotations](#use-the-human-annotation-scorer-in-the-ui), and [modify your annotation scorers using the API](#modify-a-human-annotation-scorer-using-the-api).
 
-Human annotations are supported in the weave UI after configuration of a Human Annotation scorer, which can be accessed via the `Scorers` page in the navigation sidebar. Once configured, annotations can be used when inspecting individual calls in the main Call or Evaluation table, by selecting the marker icon in the call header (seen below).
+### Create a human annotation scorer in the UI
 
-![Marker icon in call header](./imgs/marker-icon.png)
+To create a human annotation scorer in the UI, do the following:
 
-### Creating a Human Annotation scorer
+1. In the sidebar, navigate to **Scorers**.
+2. In the upper right corner, click **+ Create scorer**.
+3. In the configuration page, set:
+   - `Scorer type` to `Human annotation`
+   - `Name`
+   - `Description`
+   - `Type`, which determines the type of feedback that will be collected, such as `boolean` or `integer`.
+4. Click **Create scorer**. Now, you can [use your scorer to make annotations](#use-the-human-annotation-scorer-in-the-ui).
 
-To create a scorer, click "create scorer" in the "Scorers" page (accessed via the navigation sidebar). Select the type of scorer, in this case: "Human annotation". Then fill out the subsequent form to configure the scorer, paying special attention to the `Type`, which will be used to determine the type of feedback that will be collected. Here is an example scorer configuration where a human labeler is asked to choose which type of document the llm used:
+In the following example, a human annotator is asked to select which type of document the LLM ingested. As such, the `Type` selected for the score configuration is an `enum` containing the possible document types.
```

Comment:
not sure what we want to do about this but currently in the UI we show "options" as the label for "enum". It is still "enum" in code, but that was a bit technical for the UI...